.github/workflows/workflow.yaml

```diff
@@ -466,6 +530,7 @@ jobs:
           files: |
             _dist/sha256sums.txt
             _dist/*.tar.gz
+            _dist/release_benchmarks.txt
```

Comment:
do we really want to publish benchmark results for the release now? I think this could be useful, but per Max's comments above, if we only compare the benchmark run against tip of main/ base of branch then would this be actually used?

Reply:
Yeah, if we are considering running a benchmark for head every time, then we don't need to run/publish benchmark in the release.